### DISCUSSION TITLE
No osmium ore generated?

### DISCUSSION THREAD

**Discussion by haslilsah** [NillerMedDild/Enigmatica6#1965] [Q&A]
Hi there! 

I've been playing Enigmatica 6 for a while now and started to get into the mekanism modpack. 

Pretty quick, to make advancements, you need osmium ingots. 
My issue now is that, no matter how long and where I search, there doesn't seem to be any osmium ore generating... 

Does anybody have the same problem? And if so, is there a way to solve it? 
I know osmium is supposed to be rare, but by now I should have surely found some (Basically dug out a whole chunk).

Thanks for any answers and suggestions!

**Comment by NielsPilgaard** [comment]
Hey there! 

Which version are you playing? Osmium isn't really supposed to be rare, there should be about 100 per chunk.

**Reply by haslilsah** [reply]
Hi! 
Im playing version 1.16.5 of Enigmatica 6.

Oh okay, I always thought there be less...

**Reply by NielsPilgaard** [reply]
1.16.5 is the Minecraft version, not modpack version :)

**Reply by haslilsah** [reply]
Ooh yeah... Sorry.. 😅

It's version 0.4.8

**Reply by NielsPilgaard** [reply]
I can't reproduce this at all :/ How are you searching for it?

**Reply by haslilsah** [reply]
No worries.. 

Well basically it should generate at any height in the world, but mostly between 10-40 ish blocks I think, can't remember the exact number.. 
I mined out multiple chunks on those heights and even one chunk all the way (top to bottom) and couldn't find any :/

**Reply by NielsPilgaard** [reply]
It can spawn in Y 20-44 yup, with the highest concentration being at 32.

Please head over to our Discord so we can troubleshoot further: https://discord.gg/xPPwaFx
